lib/code_corps/stripe/in_memory.ex

```diff
@@ -0,0 +1,24 @@
+defmodule CodeCorps.Stripe.InMemory do
+  defmodule Customer do
+    def create(map) do
+      {:ok, do_create(map)}
+    end
+
+    defp do_create(_) do
+      {:ok, created} = DateTime.from_unix(1479472835)
+
+      %Stripe.Customer{
+        id: "cus_9aMOFmqy1esIRE",
+        account_balance: 0,
+        created: created,
+        currency: "usd",
+        default_source: nil,
+        delinquent: false,
+        description: nil,
+        email: "mail@test.com",
+        livemode: false,
+        metadata: %{}
+      }
+    end
```

Comment:
I'm not sure this has gone far enough. In http://blog.plataformatec.com.br/2015/10/mocks-and-explicit-contracts/ it looks like the recommendation is to use behaviours and callbacks. That is, we seem to be lacking explicit contracts right now.

The boundary of testing the HTTP interactions should probably happen at the level of `stripity-stripe` and not here, so I don't think that will have to be added. But even in adjusting the small cases we have here, I ended up with the behavior getting out of sync because the contract is not explicit.